lighthouse-core/audits/dobetterweb/js-libraries.js

```diff
@@ -32,6 +32,7 @@ class JsLibrariesAudit extends Audit {
     return {
       id: 'js-libraries',
       title: str_(UIStrings.title),
+      scoreDisplayMode: Audit.SCORING_MODES.INFORMATIVE,
```

Comment:
I think you need to do n/a on passing to hide this now.

Reply:
ah shoot, yes great catch thank you!

Reply:
sidenote for future: we really should consider automatically do this for informative audits when score=1 :)